mutis/signal.py

```diff
@@ -112,84 +133,127 @@ def OU_fit(self, bins=None, rang=None, a=1e-5, b=100):
             plt.plot(x_c, self.pdf(x_c, *popt), "k-", label="curve_fit", alpha=0.8)
             res["curve_fit"] = (popt, np.sqrt(np.diag(pcov)))
         except Exception as e:
-            raise Exception(f"Some error fitting with curve_fit {e}")
+            popt, pcov = None, None
```

Comment:
remove this line, not needed

Reply:
the `Python popt, pcov = None, None`?

Reply:
In this function `res` is always returned, so I guess you always expect to have the dict `res` returned. Then, after calling this function, if you access the expected returned dict, you also expect it to have the keywords `curve_fit`. `MLE_fit`, `th_est1` and `th_est2`, with values or empty if errors are logged. 

This is not the case with `popt` and `pcov`, they are not returned by the function. So, I would just remoce this single line.

Reply:
Oh thats right, thanks. Done!